src/app/ApplicationTemplate.Shared.Views/App.xaml.cs

```diff
@@ -31,33 +29,46 @@ public App()
 
 	public Shell Shell { get; private set; }
 
+#if false
 	public MultiFrame NavigationMultiFrame => Shell?.NavigationMultiFrame;
+#endif
 
-	public Window CurrentWindow => Windows.UI.Xaml.Window.Current;
+	public Window CurrentWindow { get; private set; }
 
-	protected override void OnLaunched(LaunchActivatedEventArgs args)
+	protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
 	{
-		InitializeAndStart(args);
+		InitializeAndStart();
 	}
 
+	/*
 	protected override void OnActivated(IActivatedEventArgs args)
 	{
 		// This is where your app launches if you use custom schemes, Universal Links, or Android App Links.
 		InitializeAndStart(args);
 	}
+	*/
 
-	private void InitializeAndStart(IActivatedEventArgs args)
+	private void InitializeAndStart(IActivatedEventArgs args = null)
 	{
+#if WINDOWS
+		CurrentWindow = new Window();
+		CurrentWindow.Activate();
```

Comment:
I don't think you need to activate now considering that you already do it at line 80.